velox/functions/prestosql/StringFunctions.cpp

```diff
@@ -275,6 +275,160 @@ class ConcatFunction : public exec::VectorFunction {
   std::vector<StringView> constantStringViews_;
 };
 
+/**
+ * concat_ws(separator, string1, ..., stringN) → varchar
+ * Returns the concatenation of string1, string2, ..., stringN connected by
+ * separator.
+ * */
+class ConcatWsFunction : public exec::VectorFunction {
```

Comment:
Please move it to Spark folder

Reply:
If it both exists in Presto and Spark, please update the PR title and PR description